### DISCUSSION TITLE
Is it safe to use this mod in a server？

### DISCUSSION THREAD

**Discussion by ghost** [cc-tweaked/CC-Tweaked#1509] [Q&A]
Is it safe to use this mod in a server？

**Comment by ghost** [comment]
thanks

**Comment by SquidDev** [comment]
**TLDR:** It depends, but most of the time yes.

ComputerCraft is sandboxed and imposes various rate limits on the code run, which means you shouldn't need to worry about players breaking your system or causing lag with ComputerCraft. Lots of servers have run this mod over the years who haven't had issues.

_That all said_, I can't guarantee that there aren't bugs in the mod which could cause problems. If you've got players who are actively trying to crash the server I probably wouldn't recommend CC. But also, I'm not sure there's many mods I could recommend in that case!
